### DISCUSSION TITLE
Medley on Crostini Linux: X vs SDL performance

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1441] [General]
On my Chromebox with Crostini Linux, the Linux container of chromeOS based on Debian 11 Bullseye, I built Medley and the full loadup with apps from the latest sources using gcc. Since Crostini uses Wayland I was cursious to compare the performance of the X and SDL versions of Medley. Like on the Raspberry Pi 400 with Wayland (see #1378), it turns out SDL is noticeably faster.

Here is X on Crostini:

[medley-crostini-x.webm](https://github.com/Interlisp/medley/assets/10342479/56af099c-1781-4fc6-a77f-711505efbf1e)

And here SDL:

[medley-crostini-sdl.webm](https://github.com/Interlisp/medley/assets/10342479/b11c065e-af49-48c5-89c8-354cd8dcf777)

**Comment by Kirtai** [comment]
I noticed that the SDL version is also much faster than the X11 version on X.

Try building the sysouts to see a real difference.

**Reply by pamoroso** [reply]
Thanks, the SDL version is indeed faster.
